server/database/schemas/SteamGame.js

```diff
@@ -0,0 +1,51 @@
+const mongoose = require('mongoose')
+
+const SteamGamePriceSchema = new mongoose.Schema({
+  currency: {
+    type: String
+  },
+  initial: {
+    type: Number
+  },
+  final: {
+    type: Number
+  },
+  discount_percent: {
+    type: Number
+  },
+  initial_formatted: {
+    type: String
+  },
+  final_formatted: {
+    type: String
+  }
+})
+
+const SteamGameSchema = new mongoose.Schema({
```

Comment:
We'll need a `last_updated` time stamp to reference for the future when we re-run the DB scripts.